src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf"),
+    url("./fonts/Avenir-Heavy.ttf"),
+    url("./fonts/Avenir.ttc");
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form-big {
+  margin-bottom: 20px;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+}
+
+.form-small {
+  width: 100%;
+  height: 42px;
+}
+
+.bar {
+  width: 100%;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
```

Comment:
inherit font, do not repeat it

Reply:
![image](https://user-images.githubusercontent.com/70288388/191593229-769ccaa8-c2d0-4eb1-8d84-575f94bd288f.png)
readme says that is not inherits, if i delete "font-family: "Avenir", sans-serif;" from this, font is changes

Reply:
you need to use here `font-family: inherit` and move `font-family: "Avenir", sans-serif;` to the `html` selector